packages/input/Input.tsx

```diff
@@ -57,14 +57,16 @@ export class Input extends Component<InputProps> {
 
   inputElement: HTMLInputElement;
 
-  private provideValue(event: Event) {
+  private delegateEventToParent( event: Event ) {
+
     this.value = this.inputElement.value;
-    emit(this,'change'); // emit change event on root element
+    emit(this, event.type);
```

Comment:
this is wrong implementation. oninput events are propagated from shadow boundaries. by doing this we are triggering double oninput event